src/game_battler.cpp

```diff
@@ -531,7 +533,7 @@ void Game_Battler::ChangeHp(int hp) {
 
 		// Death
 		if (GetHp() <= 0) {
-			AddState(RPG::State::kDeathID);
+			AddState(RPG::State::kDeathID, true);
```

Comment:
> AddState - add allow_battle_states

Why does this recursive call use ``true```instead of ``allow_battle_states``

Reply:
Wrong function :see_no_evil:
(Easy to mess up with the omission arrows)